### PR TITLE
Dependencies and installation instructions

### PR DESCRIPTION
**Pull request by hervenicol**
Should smooth up setup for next people who want to use it.

### CODE REVIEW

**Review by jejemontagne — APPROVED** [review]
cssh is clusterssh on Ubuntu

File: README.md [hunk 1]
```diff
@@ -13,6 +13,24 @@ It's technically safe to do
 alias dsh=dtcsh
 ```
 
+However, it still relies on the original dsh.
+
+# How to install it ?
+
+## Prerequisites
+
+You need to install `zsh`, `dsh`, `rsh` (for default rsh communication) and `cssh` (for interactive shells) first of all.
```
Comment (Anvil) [inline README.md]: Please remove rsh mentions :-P It's a dsh dependency (if anything), not dtcsh's

**Comment by Anvil** [comment]
> cssh is clusterssh on Ubuntu

Same in EPEL repository

**Comment by hervenicol** [comment]
> > cssh is clusterssh on Ubuntu
> 
> Same in EPEL repository

I added a note for that.

**Pull request merged**